python/Ganga/new_tests/GPI/Bugs/Issue185.py

```diff
@@ -0,0 +1,31 @@
+from ..GangaUnitTest import GangaUnitTest
+
+class Issue185(GangaUnitTest):
```

Comment:
Also, since this doesn't use any of the Ganga runtime machinery I'd rather this was done as a standard UnitTest, not in the `GPI` directory. In a while I'm going to have to rearrange these anyway but if this is already in the right pace then it will be easier.

In fact, I would just call this `test_safe_save` and keep it as a generic unit test for the function rather than tying it to a specific issue.